products-service/functions/catalogBatchProcess.ts

```diff
@@ -0,0 +1,40 @@
+import { SQSHandler } from "aws-lambda";
+import { Client } from "pg";
+import { dbOptions } from "../config";
+
+export const catalogBatchProcess = (sns): SQSHandler => {
+    return async (event) => {
+        const client = new Client(dbOptions);
```

Comment:
Just an advice: it would be better, if you separate your DB logic into another module, so you can reuse it both here and in POST-lambda

Reply:
@spital thanks a lot, I'll edit it